From: Takashi Iwai <tiwai@suse.de>
Subject: [ALSA] Correct detection of iBook G4 1420Mhz soundcard
Patch-mainline: 
References: 

Here is a patch to correct detection of the soundcard on my iBook model
(bought really recently).
Without that fix, there were only andui in the headphone, and the mixer
was totaly non functional.

Signed-off-by: Takashi Iwai <tiwai@suse.de>

---


diff -u linux/sound/ppc/pmac.c:1.53 linux/sound/ppc/pmac.c:1.54
--- linux/sound/ppc/pmac.c:1.53	Tue Aug 30 04:17:57 2005
+++ linux/sound/ppc/pmac.c	Wed Sep  7 05:28:14 2005
@@ -988,6 +988,7 @@
 		case 0x33:
 		case 0x29:
 		case 0x24:
+		case 0x5c:
 			chip->num_freqs = ARRAY_SIZE(tumbler_freqs);
 			chip->model = PMAC_SNAPPER;
 			chip->can_byte_swap = 0; /* FIXME: check this */
